app/src/main/java/com/mmm/parq/activities/DriverActivity.java

```diff
@@ -287,8 +301,16 @@ public void onSaveInstanceState(Bundle savedInstanceState) {
     }
 
     @Override
+    public void onActivityResult(int requestCode, int resultCode, Intent data) {
+        // DO NOT REMOVE -- necessary in order for onActivityResult to be called in fragment.
```

Comment:
I had a solution for this, I'll probably remember it by the time I get finished doing this code review, just writing something here so I don't forget about it.